src/Admin/AbstractAdmin.php

```diff
@@ -1044,9 +1044,16 @@ public function getTemplates()
 
     public function getTemplate($name)
     {
-        if (isset($this->templates[$name])) {
+        if ($this->hasTemplate($name)) {
             return $this->templates[$name];
         }
+
+        throw new \RuntimeException(sprintf('Template "%s" does not exist', $name));
```

Comment:
IMHO `\InvalidArgumentException` fits better for this validation.

Reply:
That's correct, this can detected just by analyzing the code: https://stackoverflow.com/questions/5586404/logicexception-vs-runtimeexception#5586510

Reply:
Updated the exception thrown.